<launch>
		
	<node pkg="bb_mpu9150" type="mpu9150_node" name="mpu9150_node"/>
	
	<node pkg="robot" type="lpp_node" name="lpp_node" 
	output="screen"/>
	
</launch>
